frontend/pages/hubs/[hubUrl].js

```diff
@@ -264,21 +266,23 @@ export default function Hub({
           <div ref={contentRef} className={classes.contentRef} />
           {!isLocationHub && <BrowseExplainer />}
           <BrowseContent
-            initialProjects={initialProjects}
-            initialOrganizations={initialOrganizations}
-            initialMembers={initialMembers}
-            filterChoices={filterChoices}
-            loadMoreData={loadMoreData}
             applyNewFilters={applyNewFilters}
             applySearch={applySearch}
-            hideMembers={!isLocationHub}
             customSearchBarLabels={customSearchBarLabels}
-            handleSetErrorMessage={handleSetErrorMessage}
             errorMessage={errorMessage}
-            hubQuickInfoRef={hubQuickInfoRef}
+            // TODO: is this still needed?
+            // filterChoices={filterChoices}
```

Comment:
This is still needed. If removed, you can't catch possibleFilters from `public/data/filterChoices.js` and will get this error: `Error: No filter choices supplied`